pkg/api/types.go

```diff
@@ -887,6 +887,11 @@ type QuobyteVolumeSource struct {
 	// Default is no group
 	// +optional
 	Group string
+
+	// Name of Quobyte Tenant that the Volume is part of
+	// Default is DEFAULT
```

Comment:
`DEFAULT` because it's the default tenant if no tenant is supplied (and `DEFAULT` is just the name for the default tenant). I actually don't think that a link to the docs would be useful but I can add a little description like above to the default.

This is intended for persistent volume sources (and should be set during the dynamic provisioning process). If a user can use a `QuobyteVolume` in a pod spec he is able to choose any tenant.

@quolix any other thoughts on this?